Package.swift

```diff
@@ -10,15 +10,15 @@ let package = Package(
     ],
     products: [
         .library(name: "Apodini", targets: ["Apodini"]),
-        .library(name: "ApodiniVaporSupport", targets: ["ApodiniVaporSupport"]),
-        .library(name: "ApodiniREST", targets: ["ApodiniREST"]),
+        .library(name: "ApodiniDatabase", targets: ["ApodiniDatabase"]),
         .library(name: "ApodiniGRPC", targets: ["ApodiniGRPC"]),
+        .library(name: "ApodiniJobs", targets: ["ApodiniJobs"]),
+        .library(name: "ApodiniNotifications", targets: ["ApodiniNotifications"]),
         .library(name: "ApodiniOpenAPI", targets: ["ApodiniOpenAPI"]),
-        .library(name: "ApodiniWebSocket", targets: ["ApodiniWebSocket"]),
         .library(name: "ApodiniProtobuffer", targets: ["ApodiniProtobuffer"]),
-        .library(name: "ApodiniDatabase", targets: ["ApodiniDatabase"]),
-        .library(name: "ApodiniNotifications", targets: ["ApodiniNotifications"]),
-        .library(name: "ApodiniJobs", targets: ["ApodiniJobs"])
+        .library(name: "ApodiniREST", targets: ["ApodiniREST"]),
+        .library(name: "ApodiniVaporSupport", targets: ["ApodiniVaporSupport"]),
+        .library(name: "ApodiniWebSocket", targets: ["ApodiniWebSocket"])
```

Comment:
You might want to make the `ApodiniTypeReflection` library available here for consumers of the Apodini package